setup.py

```diff
@@ -48,22 +48,30 @@
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     install_requires = [
-        'Django==1.8.13',
-        'django-autocomplete-light==2.3.3',
-        'django-extensions>=1.5.5',
-        'django-nose>=1.4',
+        'Django>=1.8.2',
```

Comment:
> I'm afraid that some code of this project is not compatible with Django 1.9 or 1.10

☹️  All I really want in this project is the django-powerdns-dnssec app, and I'm using Django 1.10. But I get where your coming from and I can work around it by explicitly installing `Django==1.10` after installing `requirements.txt` (after D-P-D gets into pip-installable shape for my uses).

Instead of pinning it to a specific Django patch level, would you accept `Django>=1.8,<1.9` so we can get the most recent patchlevel of Django 1.8?